docs/products/flink/concepts/managed-service-features.rst

```diff
@@ -0,0 +1,44 @@
+Aiven for Apache Flink® managed service features
+================================================
+
+As a fully managed distributed data stream processing platform, deployable in the cloud of your choice, some of the key features of a managed Flink service include the following:
+
+Cluster deployment mode
+-----------------------
+Aiven for Apache Flink® is configured to use the `HashMap state backend <https://ci.apache.org/projects/flink/flink-docs-stable/api/java/org/apache/flink/runtime/state/hashmap/HashMapStateBackend.html>`_. This means that the `state <https://ci.apache.org/projects/flink/flink-docs-release-1.15/docs/concepts/stateful-stream-processing/#what-is-state>`_ is stored in memory, which can impact the performance of jobs that require keeping a very large state. We recommend you provision your platform accordingly.
+
+The Flink cluster executes applications in `session mode <https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/deployment/overview/#session-mode>`_ so you can deploy multiple Flink jobs on the same cluster, thus effectively utilizing the available resources.
+
+Scaling a Flink cluster
+-----------------------
+Each node is equipped with a TaskManager and JobManager. We recommend scaling up your cluster to add more CPU and memory for the TaskManager before attempting to scale out, so you make the best use of the resources with a minimum number of nodes. 
+
+By default, each TaskManager is configured with a single slot for maximum job isolation. It is highly recommended that you modify this option to match your requirements.
+
+.. note:: Adjusting the task slots per TaskManager requires a cluster restart.	
+
+Cluster restart strategy
+------------------------
+The cluster's default restart strategy is configured to Failure Rate. This controls Flink's restart behavior in cases of failures during the execution of jobs. Administrators can overwrite this setting in the service's advanced configuration options.
```

Comment:
What does Failure rate means? Is there a link to understand more?
```suggestion
The cluster's default restart strategy is configured to `Failure Rate`. This controls Flink's restart behavior in cases of failures during the execution of jobs. Administrators can overwrite this setting in the service's advanced configuration options.
```

Reply:
Fixed.